foundation/plugins/org.jboss.tools.foundation.core/src/org/jboss/tools/foundation/core/credentials/internal/CredentialDomain.java

```diff
@@ -60,7 +60,7 @@ public CredentialDomain(Preferences pref) throws BackingStoreException {
 
 		credentials = new HashMap<String, String>();
 		String usersList = pref.get(PROPERTY_USER_LIST, (String)null);
```

Comment:
I suppose that would have worked as well, but I prefer to actually know whether the string was null (ie not set) or empty (ie previously persisted as an empty list) so that I can alter behavior based on those two conditions if I need to later.  Anyway, it's already been pushed.